crates/cext/src/dag.rs

```diff
@@ -194,30 +194,30 @@ pub unsafe extern "C" fn qk_dag_free(dag: *mut DAGCircuit) {
 /// Convert a given DAG to a circuit.
 ///
 /// @param dag A pointer to the DAG from which to create the circuit.
-/// @param copy_operations Performs a deep copy of operations if set to true.
 ///
-/// @return A pointer to the new circuit.
+/// @return A pointer to the new circuit. The pointer needs to be freed with ``qk_circuit_free``.
 ///
 /// # Example
 /// ```c
 /// QkDag *dag = qk_dag_new();
 /// QkQuantumRegister *qr = qk_quantum_register_new(2, "qr");
 /// qk_dag_add_quantum_register(dag, qr);
-/// QkCircuit *qc = qk_dag_to_circuit(dag, false);
+/// qk_quantum_register_free(qr);
+///
```

Comment:
nit: Might be nice to remove some of these newlines, but it's personal preference.